src/components/PersonalInfo.tsx

```diff
@@ -0,0 +1,44 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import useTranslationHook from '../hook/TranslationHook'
+
+interface PersonalInfoProps {
+  profile: {
+    first_name: string
+    last_name: string
+    email: string
+  }
```

Comment:
Why not using Profile from src/types/index.ts? Would typescript make it annoying because the consuming component doesn't have a full profile available?

Reply:
replace by Profile from src/types/index.ts